### DISCUSSION TITLE
Import SW5 Blog to OpenBlogware

### DISCUSSION THREAD

**Discussion by restegit** [Werkstattl/OpenBlogware#11] [General]
Is it possible to import all blogposts from a Shopware 5 system to a new Shopware 6 with the OpenBlogware plugin installed?

**Comment by 7underlines** [comment]
Currently, there isn’t an out-of-the-box solution in OpenBlogware to import blog posts directly from Shopware 5 to Shopware 6 due to differences in the database structure between the two versions. OpenBlogware doesn’t include a dedicated import functionality for this migration.

However, it is possible to achieve this with a custom script. Here is how I would approach this:

1. Access the Shopware 5 API to read the existing blog posts.
2. Use the Shopware 6 API to create corresponding blog entries in your Shopware 6 system.

**Reply by restegit** [reply]
Thanks a lot.
